apps/nextra/pages/en/_meta.ts

```diff
@@ -16,7 +16,7 @@ export default {
         title: "Get Started",
       },
       "smart-contracts": {
-        title: "Smart Contracts",
+        title: "Smart Contracts (⚠️)",
```

Comment:
Why are we adding a warning sign?

Reply:
Just want to warn people its under construction